src/client/src/commons/form/borehole/stratigraphy/hierarchicalDataSearch.js

```diff
@@ -95,6 +95,8 @@ const HierarchicalDataSearch = ({ schema, labels, selected, onSelected }) => {
             ),
           });
         }
+
+        schemaData.sort((a, b) => a.order - b.order);
```

Comment:
REQ: Hier wird schemaData in place sortiert. Die Daten, die vom `useDomainSchema` kommen sollten aber nicht verändert werden, da react-query diese cached und andere Komponenten, diese ebenfalls verwenden könnten.
Mit `schemaData.slice().sort(...` kann vor dem sortieren einfach eine kopie des arrays gemacht werden.